### PR TITLE
Refer to the section where the deployment steps are described

### PR DESCRIPTION
**Pull request by mnmami**
Valuable but hidden information. I think the description of the steps should be referred to from the top if not entirely moved there.

### CODE REVIEW

File: content/en/docs/gke/deploy/deploy-cli.md [hunk 1]
```diff
@@ -33,6 +33,9 @@ one if you haven't already.
   IAP)](https://cloud.google.com/iap/docs/).
 
 
+Refer to [Understanding the deployment process 
+](#understanding-the-deployment-process) for a description about the deployment steps and artifacts.
```
Comment (8bitmp3) [inline content/en/docs/gke/deploy/deploy-cli.md:37]: Suggestion:

- Small refactoring to replace "for a description of the process" bit, which looks a little bit "off"

```diff
- Refer to 
- [Understanding the deployment process](#understanding-the-deployment-process)
- for a description about the deployment steps and artifacts.
+ To understand the kfctl configuration and deployment process in more detail, 
+ refer to
+ [Understanding the deployment process](#understanding-the-deployment-process).
```

or

```diff
Refer to 
[Understanding the deployment process](#understanding-the-deployment-process)
- for a description about the deployment steps and artifacts.
+ for more information on the kfctl configuration and deployment process.
```

(or something along these lines with/without mentioning "artifacts")

**Review by 8bitmp3 — CHANGES_REQUESTED** [review]
Hey @mnmami, thanks for the PR! Some suggestions from my side:

- Small refactoring to replace "for a description of the process" bit, which looks a little bit "off"

```diff
- Refer to 
- [Understanding the deployment process](#understanding-the-deployment-process)
- for a description about the deployment steps and artifacts.
+ To understand the kfctl configuration and deployment process in more detail, 
+ refer to
+ [Understanding the deployment process](#understanding-the-deployment-process).
```

or

```diff
Refer to 
[Understanding the deployment process](#understanding-the-deployment-process)
- for a description about the deployment steps and artifacts.
+ for more information on the kfctl configuration and deployment process.
```

(or something along these lines with/without mentioning "artifacts")

Thank you

File: content/en/docs/gke/deploy/deploy-cli.md (hunk 1 above)
Comment (mnmami) [reply content/en/docs/gke/deploy/deploy-cli.md:37]: Both look good to me.

Comment (8bitmp3) [reply content/en/docs/gke/deploy/deploy-cli.md:37]: @mnmami Ok! Please do go ahead and make the changes, then we can `ok-to-test` it and move forward. Thanks!

Comment (8bitmp3) [reply content/en/docs/gke/deploy/deploy-cli.md:37]: @mnmami Hi! Let us know if you have any updates like commits with new changes, etc 👍

Comment (mnmami) [reply content/en/docs/gke/deploy/deploy-cli.md:37]: @8bitmp3 I don't have more changes here. The above LGTM.

Comment (8bitmp3) [reply content/en/docs/gke/deploy/deploy-cli.md:37]: Thanks for getting back @mnmami. I can't make changes to your PR, I think. Can you please make a few revisions, stage and push and we'll take a look?

Comment (mnmami) [reply content/en/docs/gke/deploy/deploy-cli.md]: @8bitmp3 sure, done.

**Comment by 8bitmp3** [comment]
/ok-to-test

also

/assign @joeliedtke @Bobgy 

PTAL thank you!

**Comment by Bobgy** [comment]
Thanks!
All the kfctl mentions are in fact outdated, but we can fix it in a separate PR

**Pull request merged**